### ISSUE TITLE
Nested parameters

### ISSUE THREAD

**Issue by ctusch** [tgreyuk/typedoc-plugin-markdown#191]
I can't seem to get nested parameters to work. Without the plugin (i.e. normal HTML output) it does work. I have found a related bug is TypeDoc (TypeStrong/typedoc#987) but it has been fixed a year ago.

Here is an example:
```typescript
	/**
	 * Creates a new group.
	 * @param params The parameters passed to the method.
	 * @param params.name The name of the new group.
	 * @param context The context of the method call.
	 */
	@method()
	async create(params: { name: string; parent?: number }, context: CallContext) {
```

The second `@param` is not output at all.

**Comment by tgreyuk** [comment]
Yeah, the issue it's because the params are in a table display and it's not possible to nest in pure markdown. The solution I think is to expand the params (as per example) when nested params exists:

![image](https://user-images.githubusercontent.com/11680870/106519629-80feee80-64d3-11eb-9cdb-600ebe2408d9.png)

**Comment by ctusch** [comment]
But what would happen if the nesting went deeper? Just indent the list and use tables at the leaf nodes? I'm not sure how this would look. Maybe another solution would be to keep the single table and make the list flat with qualified names (like in the JSDoc source itself). Or maybe introduce a CLI argument to use nested lists _instead_ of tables?

**Comment by tgreyuk** [comment]
Yes this would only work for one level of nesting so doesn't completely solve the problem. List with qualified names might work will give it a try. Nested lists with the restrictions of markdown formatting and readability might be problematic  (i have tried something similar in the past and gave up but might be worth revisiting). Thanks for the ideas.

**Comment by tgreyuk** [comment]
@ctusch implemented a flat structure as like:

![image](https://user-images.githubusercontent.com/11680870/107996453-5aaf7780-6fd8-11eb-895f-705445fa69fc.png)

I think this is the best solution. Available in 3.5.0.

**Comment by ctusch** [comment]
Thank you, that works beautifully!
